README.md

```diff
@@ -19,11 +19,11 @@ Collect Ansible roles for automatic deployment.
 
 	**Playbook Example:**
 	```yaml
-  ---
-  - hosts: servers
-    roles:
-      - { role: dropwizard-api, become: yes }
-    	- { role: hubot, become: yes }
+      ---
+      - hosts: servers
+        roles:
+          - { role: dropwizard-api, become: yes }
+        	- { role: hubot, become: yes }
 	```
```

Comment:
The ansible version and the badge of this file need to be updated to 2.8.1 as well.